libsoftwarecontainer/include/container.h

```diff
@@ -75,8 +75,7 @@ class Container :
      * @param shutdownTimeout Timeout for shutdown of container.
      */
     Container(
-            const std::string &id
-            , const std::string &name
+            const std::string id
```

Comment:
Yes. Container holds this value now instead of SoftwareContainer since SoftwareContainer does not use it.